docs/detections/rules-ui-monitor.asciidoc

```diff
@@ -31,9 +31,13 @@ and then reactivating them in small batches at staggered intervals. This
 ensures {kib} does not attempt to run all the rules at the same time.
 * Consider adding another {kib} instance to your environment.
 
-Even if your rule runs at its scheduled time, there might still be missing alerts if your ingestion pipeline delay is greater than your rule interval + additional look-back time. As such, it is recommended that you use caution when creating custom rules with interval + additional look-back time values smaller than 6 minutes.
+Even if your rule runs at its scheduled time, there might still be missing alerts if your ingestion pipeline delay is greater than your rule interval + additional look-back time. Prebuilt rules have a minimum interval + additional look-back time of 6 minutes in {stack} version >=7.11.0. To avoid missed alerts for prebuilt rules, use caution to ensure that ingestion pipeline delays remain below 6 minutes.
 
-You can also eliminate ingestion pipeline delay by specifying the `Timestamp override` field value to `event.digested` in <<rule-ui-advanced-params, advanced settings>> during rule creation. The detection engine uses the value from the `event.digested` field as the timestamp of when to execute the rule. For example, if an event occurred at 10:01 and was ingested in {es} at 10:10, the rule you created to detect that event would execute at 10:10 and generate an alert. 
+In addition, use caution when creating custom rule schedules to ensure that the specified interval + additional look-back time is greater than your deployment's ingestion pipeline delay.
+
+You can eliminate the risk of missed alerts due to ingestion pipeline delay by specifying the `Timestamp override` field value to `event.digested` in <<rule-ui-advanced-params, advanced settings>> during rule creation or editing. The detection engine uses the value from the `event.digested` field as the timestamp when executing the rule.
```

Comment:
`event.digested` should be `event.ingested` 2x

Otherwise LGTM, thanks!

Reply:
Ah, good catch - thank you!

Reply:
although `event.digested` seems like a cool name for future field :-)